packages/components/src/CountryDropDown.purs

```diff
@@ -11,27 +12,38 @@ import React.Basic.DOM.Events (targetValue)
 import React.Basic.Events (handler)
 
 
-
 countryDropDown :: Array DropDownCountry -> Boolean -> (Maybe String -> Effect Unit) -> Maybe String -> JSX
 countryDropDown countries disabled onChange value =
   DOM.div
     { className: "input-field--container"
     , children:
         [ inputLabel { label: "Land", nameFor: "country" }
         , DOM.select
-            { children: map createOption countries
+            { children: cons placeholderOption $ map createOption countries
             , onChange: handler targetValue onChange
-            , value: fromMaybe "FI" value
```

Comment:
I think it's not right to ignore the `value` parameter like this. Instead, `value` should be set to the placeholder value when its corresponding state is initialized in React. The full life cycle of forms fields with React should have functions also for taking updates from the state, not only updating it.